### PR TITLE
linting and formatting of the e2e test files

### PR DESCRIPTION
**Pull request by amihajlovski**
## Summary
Describe the changes proposed in this pull request:
- What is the motivation for this change?
Formatting the e2e files
- What existing problem does this pull request solve?
Better development practise

## Tested scenarios
Description of tested scenarios: 
- E2E tests

**Fixed issue**:  SFI-710

### CODE REVIEW

File: tests/playwright/fixtures/countriesEUR/FR.spec.mjs [hunk 1]
```diff
@@ -60,7 +60,9 @@ test.describe.parallel(`${environment.name} EUR FR`, () => {
     }
     redirectShopper = new RedirectShopper(page);
     const result = await redirectShopper.doAmazonPayment();
-    if(result != true){test.skip()};
+    if (!result) {
+      test.skip()
```
Comment (dcardos) [inline tests/playwright/fixtures/countriesEUR/FR.spec.mjs:64]: Should this finish with ";"?

File: tests/playwright/fixtures/USD.spec.mjs [hunk 2]
```diff
@@ -264,12 +264,11 @@ for (const environment of environments) {
         redirectShopper = new RedirectShopper(page);
         await redirectShopper.doPayPalPayment(true, false, true);
         if (environment.name.indexOf('v5') !== -1) {
-	    await page.locator("button[value='place-order']").click();
+	          await page.locator("button[value='place-order']").click();
```
Comment (dcardos) [inline tests/playwright/fixtures/USD.spec.mjs]: Maybe it was a bug in github, but now the identation seems odd here.

File: tests/playwright/fixtures/USD.spec.mjs [hunk 3]
```diff
@@ -280,11 +279,10 @@ for (const environment of environments) {
...
             await page.locator("button[value='place-order']").click();
-	    await page.locator(".order-thank-you-msg").isVisible({ timeout: 20000 });
-        }
-	else {
-	    await checkoutPage.expectSuccess();
-	}
+	          await page.locator(".order-thank-you-msg").isVisible({ timeout: 20000 });
+        } else {
+	        await checkoutPage.expectSuccess();
+	      }
```
Comment (dcardos) [inline tests/playwright/fixtures/USD.spec.mjs]: Same here, the awaits seems not aligned.

**Review by dcardos — CHANGES_REQUESTED** [review]
Maybe indentation issue and maybe missing ";"

**Review by zenit2001 — APPROVED** [review]

**Pull request merged**